platformio.ini

```diff
@@ -31,13 +31,11 @@ board_build.partitions = min_spiffs.csv
 lib_deps =
     ; https://arduinojson.org/v6/api/
     bblanchon/ArduinoJson @ ^6.17.2
-    ; CircularBuffer by AgileWare
-    1796
-    ; TinyGPSPlus by Mikal Hart
-    1655@1.0.2
+    rlogiacco/CircularBuffer @ ^1.3.3
```

Comment:
Whats the reason for changing the CircularBuffer Source? Is there any difference in behavior?

Reply:
This is no change - this is the same lib but not pulling it by numeric Id but the descriptive name.

Reply:
Ahh nice ok, looked different on the first view if you compare the lib name in the comment and in the link.

Reply:
Yes you are right.... I did a search for the number in platformIo and it came up with this name.